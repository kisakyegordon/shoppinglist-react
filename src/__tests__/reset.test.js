import React from 'react';
import {shallow} from 'enzyme';
import {shallowToJson} from 'enzyme-to-json';
import Reset from '../components/reset';

let match = {
    params : '1'
}

const setUp = () => {
    const props = {
        match
    };
    return shallow(<Reset{...props} />);
};


describe('<Reset />', () => {
    it('match a snapshot', () => {
        const wrapper = setUp();
        expect(shallowToJson(wrapper)).toMatchSnapshot();
    });

});