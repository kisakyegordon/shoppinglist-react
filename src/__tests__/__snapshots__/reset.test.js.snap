// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Reset /> match a snapshot 1`] = `
<div
  style={
    Object {
      "margin": "auto",
      "maxWidth": 700,
      "width": "100%",
    }
  }
>
  <Stepper
    activeStep={0}
    connector={<pure(StepConnector) />}
    linear={true}
    orientation="horizontal"
  >
    <Step>
      <StepLabel>
        Enter Your Email
      </StepLabel>
    </Step>
    <Step>
      <StepLabel>
        Answer Your Secret Question
      </StepLabel>
    </Step>
    <Step>
      <StepLabel>
        Enter Your New Password
      </StepLabel>
    </Step>
  </Stepper>
  <div
    className="signin"
  >
    <div
      style={
        Object {
          "margin": "0 16px",
        }
      }
    >
      <div>
        <p>
          <TextField
            disabled={false}
            floatingLabelFixed={false}
            floatingLabelText="Email"
            fullWidth={false}
            multiLine={false}
            name="email"
            onChange={[Function]}
            rows={1}
            type="text"
            underlineShow={true}
            value=""
          />
        </p>
        <div
          className="stepper"
          style={
            Object {
              "marginTop": 12,
            }
          }
        >
          <FlatButton
            disabled={true}
            fullWidth={false}
            label="Back"
            labelPosition="after"
            labelStyle={Object {}}
            onClick={[Function]}
            onKeyboardFocus={[Function]}
            onMouseEnter={[Function]}
            onMouseLeave={[Function]}
            onTouchStart={[Function]}
            primary={false}
            secondary={false}
            style={
              Object {
                "marginRight": 12,
              }
            }
          />
          <RaisedButton
            disabled={false}
            fullWidth={false}
            label="Next"
            labelPosition="after"
            onClick={[Function]}
            primary={true}
            secondary={false}
          />
        </div>
      </div>
    </div>
  </div>
</div>
`;
